### ISSUE TITLE
'Monitoring' recent additions and editing

### ISSUE THREAD

**Issue by karindr** [hbz/oerworldmap#762]
How can we from the oerworldmap team have a look on what is new on the map in addition to the 'recent additions' list? 
Do we want to make these insight possibilities (if there are some ;-) visible for all?

**Comment by literarymachine** [comment]
Entries sorted by creation date: https://oerworldmap.org/resource/?sort=dateCreated:desc
Entries sorted by modification date: https://oerworldmap.org/resource/?sort=dateModified:desc

I just added a route to expose the full log, accessible only by country champions and admins. Do far you simply have to know the URL as there is no link anywhere in the UI:

https://beta.oerworldmap.org/log/

In addition to that, we have talked about exposing editorial activities e.g. via twitter or email feeds etc, but I cannot find a ticket so far that manifests a decision.
